tools/docker-compose/start_tests.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
 set +x
 
-cd /awx_devel
```

Comment:
I don't actually plan on changing this file. This is just to test things without having to update the zuul config.